unity/src/test/scala/io/delta/unity/UCCatalogManagedTestUtils.scala

```diff
@@ -19,17 +19,43 @@ package io.delta.unity
 import java.net.URI
 import java.util.Optional
 
+import io.delta.kernel.commit.CommitMetadata
 import io.delta.kernel.defaults.utils.TestUtils
+import io.delta.kernel.internal.actions.{Metadata, Protocol}
 import io.delta.kernel.internal.util.FileNames
+import io.delta.kernel.test.ActionUtils
 import io.delta.storage.commit.Commit
 
 import org.apache.hadoop.fs.{FileStatus => HadoopFileStatus, Path}
 
-trait UCCatalogManagedTestUtils extends TestUtils {
+trait UCCatalogManagedTestUtils extends TestUtils with ActionUtils {
   val fakeURI = new URI("s3://bucket/table")
+  val baseTestTablePath = "/path/to/table"
+  val baseTestLogPath = "/path/to/table/_delta_log"
+
+  def createCommitMetadata(
+      version: Long = 1L,
```

Comment:
why is the default version 1?

Reply:
Does it make sense to have 2 helpers, 1 new table (version = 0, readProtocolOpt/readMetadataOpt are empty, newProtcol/newMetadata are required) and 1 for existing table (version = 1, readProtocolOpt/readMetadataOpt are required, new are optional)?

Not sure if that makes sense with how it's going to be used in tests.

Reply:
Great question/observation. 

I'm trying to have minimal test overhead / util bloat. I think for the case of commit, callers should just be sure to pass in a version. Alternatively, we could just make the version required and have everyone pass it in. I'd prefer the former option for now? i.e. what is currently present? wdyt?

Reply:
I think version should just be required then. I feel like defaults make sense when there's a reason it's the default, version 1 seems like it's a random version to me (i.e. why not 0 or 2).

Or maybe committing version 1 is the common-case and I'm just unaware?

Reply:
I'll make it required 👍